packages/grid/data-grid/src/tests/control.DataGrid.test.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import {
+  createClientRenderStrictMode,
+  // @ts-ignore
+  fireEvent,
+} from 'test/utils';
+import { expect } from 'chai';
+import { XGrid, GridComponentProps } from '@material-ui/x-grid';
+import { getCell, getRow } from 'test/utils/helperFn';
+import { spy } from 'sinon';
+import { GridSelectionModel } from '../../../_modules_/grid/models/gridSelectionModel';
+
+const isJSDOM = /jsdom/.test(window.navigator.userAgent);
+
+describe('<DataGrid /> - Control', () => {
```

Comment:
This seems out of place. If you are testing the selection, shouldn't be moved to the selection test file?

Reply:
depends if you consider that you test the control feature or the selection

Reply:
I would vote for testing the selection feature in-depth, as a end-to-end stress test, but not strong point of view.